Nos/Assets/Localization/Localizable.xcstrings

```diff
@@ -8276,6 +8293,23 @@
         }
       }
     },
+    "posts" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Posts"
```

Comment:
One more thing I just noticed: according to https://github.com/planetary-social/nos/issues/756#issuecomment-1874511286, this should be Notes, rather than Posts. Has something changed since that comment?

Reply:
No, you're right